l5kit/Pipfile

```diff
@@ -0,0 +1,42 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pytest = ">=5.4.3"
+mypy = ">=0.782"
+pytest-cov = ">=2.10.0"
+flake8 = ">=3.8.3"
+isort = ">=5.0.4"
+sphinx = ">=3.1.1"
+sphinx-rtd-theme = ">=0.5.0"
+recommonmark = ">=0.6.0"
+pre-commit = ">=2.5.1"
+black = "==19.10b0"
+importlib-metadata = "*"  # required for python 3.8
+
+[packages]
+zarr = "*"
+numpy = "*"
+opencv-contrib-python-headless = "*"
+pymap3d = "*"
+scipy = "*"
+tqdm = "*"
+transforms3d = "*"
+pyyaml = "*"
+matplotlib = "*"
+imageio = "*"
+notebook = "*"
+ptable = "*"
+ipywidgets = "*"
+protobuf = ">=3.12.2"
+torch = ">=1.5.0,<1.6.0"
+torchvision = ">=0.6.0,<0.7.0"
+ipython = "<=7.16.1"
+
+[requires]
+python_version = "3.8"
+
+[pipenv]
+allow_prereleases = true
```

Comment:
no need for this since you pinned the black version and nothing else looks like a pre-release

Reply:
It's autogenerated, should I still remove it?

Reply:
How are you autogenerating the Pipfile?

Reply:
No sorry I mean those lines were autogenerated after installing black.

Reply:
no need for that since you specify manually the black version as a pre release